### DISCUSSION TITLE
Is it possible to run Synthea only using custom made modules?

### DISCUSSION THREAD

**Discussion by mikad98** [synthetichealth/synthea#1126] [Q&A] [answered]
Is there a possibility to deactivate all modules that currently come with synthea (the 75 modules and 93 submodules up to now) and to only run synthea based on custom-made modules? 

In my use case, I need to synthesize cancer register data, which should have the same statistical properties as the original dataset (only specific cancer types and treatments that I'd define in a custom-made module).  Therefore, I wonder if that'd be possible using the synthea patient-population generator or if that's a use-case that doesn't fit syntheas purposes.

**Comment by jawalonoski** [comment, marked as answer]
Yes, it is possible. But it might not behave the way you expect, especially if you make assumptions about how Synthea works.

You can just delete all the JSON module files that come with Synthea, and then they will not run. There are also hard-coded Java modules that will continue to execute however, such as the Lifecycle module. You can see the list of built-in Java modules [here](https://github.com/synthetichealth/synthea/tree/master/src/main/java/org/mitre/synthea/modules).

Alternatively, if you are very brave and willing to explore strange new worlds alone and in the dark, you might look at [The M Feature](https://github.com/synthetichealth/synthea/wiki/The--M-Feature).
